dbot/dbot/Program.cs

```diff
@@ -25,9 +26,11 @@ public async Task MainAsync()
 
             client = new DiscordSocketClient();
             commands = new CommandService();
-            services = new ServiceCollection()
```

Comment:
there is now a private variable named serviceCollection = new ServiceCollection() that replaces this part